### PR TITLE
[rahash2] Add support for the Blowfish cipher

### PR DESCRIPTION
**Pull request by therealkbhat**
This is in reference to #4254.

Please do point out any errors and make suggestions for improvement!

### CODE REVIEW

File: libr/crypto/p/crypto_blowfish.c [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+	.use = blowfish_use,
+	.update = update,
+	.final = final
+};
+
+#ifndef CORELIB
+struct r_lib_struct_t radare_plugin = {
+	.type = R_LIB_TYPE_CRYPTO,
+	.data = &r_crypto_plugin_blowfish,
+	.version = R2_VERSION
```
Comment (XVilka) [inline libr/crypto/p/crypto_blowfish.c]: please add license here

Comment (therealkbhat) [inline libr/crypto/p/crypto_blowfish.c]: Done.

**Comment by therealkbhat** [comment]
@radare made those changes you mentioned.

Side note: I squashed my changes into the first commit; what happened to your comments? I can't see them anymore.

**Comment by radare** [comment]
Awesome! thanks!

**Comment by radare** [comment]
Yeah my comments are lost if you squash. We will miss them :___

**Pull request closed without merging**